client/src/pages/Profile/Profile.jsx

```diff
@@ -509,49 +538,136 @@ const ProfilePageLeaderPermissionDashboardLinks = () => {
           'scunt:exec show wedding missions',
           'scunt:judge bribe points',
           'scunt:judge missions',
-        ]}
-        label="Scunt Judge Panel"
-      />
-      <ProfilePageDashboardLink
-        link="/scunt-missions-dashboard"
-        authScopes={[
+        ],
+      },
+      {
+        label: 'Scunt Mission Panel',
+        anyRegisterScope: false,
+        link: '/scunt-missions-dashboard',
+        authScopes: [
           'scunt:exec show missions',
           'scunt:exec hide missions',
           'scunt:exec create missions',
           'scunt:exec delete missions',
-        ]}
-        label="Scunt Mission Panel"
-      />
-      <ProfilePageDashboardLink
-        link="/scunt-transactions"
-        authScopes={['scunt:exec view transactions']}
-        label="Scunt Point Transactions"
-      />
-      <ProfilePageDashboardLink
-        link="/scunt-game-controls"
-        authScopes={['scunt:exec game controls']}
-        label="Scunt Settings"
-      />
-      <ProfilePageDashboardLink
-        link="/faq-admin"
-        authScopes={['faq:delete', 'faq:edit']}
-        label="FAQ Admin Panel"
-      />
-      <ProfilePageDashboardLink
-        link="/timeline-admin"
-        authScopes={['timeline:create', 'timeline:edit', 'timeline:delete']}
-        label="Timeline Admin Panel"
-      />
-      <ProfilePageDashboardLink
-        link="/announcement-dashboard"
-        authScopes={['announcements:delete', 'announcements:create', 'announcements:edit']}
-        label="Announcements Admin Panel"
-      />
-      <ProfilePageDashboardLink
-        link="/frosh-info-table"
-        anyRegisterScope={true}
-        label="Frosh Info Table"
-      />
+        ],
+      },
+      {
+        label: 'Scunt Point Transactions',
+        anyRegisterScope: false,
+        link: '/scunt-transactions',
+        authScopes: ['scunt:exec view transactions'],
+      },
+      {
+        label: 'Scunt Settings',
+        anyRegisterScope: false,
+        link: '/scunt-game-controls',
+        authScopes: ['scunt:exec game controls'],
+      },
+    ],
+  };
+  return (
+    <div className={'profile-leader-dashboard-links'}>
+      <div className={'profile-leader-dashboard-permissions-links'}>
+        {leader && approved ? (
+          <Link
+            to={'/permission-request'}
+            style={{ textDecoration: 'none' }}
+            className={'no-link-style'}
+          >
+            <Button label="Request Leedur Permissions" />
+          </Link>
+        ) : (
+          <></>
```

Comment:
you'll need to put all your dropdown divs within this tag so leedurs that are not approved (i.e., `user.approved === false`) u don't see the tags yet,
![image](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/86009011/16d658d2-e1bb-4dee-8f30-f1a12832f991)